src/components/09-buttons/buttons.hbs

```diff
@@ -1,87 +1,11 @@
-            <td><button class="rvt-button button--danger rvt-button--danger-active">Button</button></td>
-            <td><button class="rvt-button button--danger" disabled>Button</button></td>
-        </tr>
-        <tr>
-            <th scope="row">Danger secondary</th>
-            <td><button class="rvt-button rvt-button--danger-secondary">Button</button></td>
-            <td><button class="rvt-button rvt-button--danger-secondary rvt-button--danger-secondary-hover">Button</button></td>
-            <td><button class="rvt-button rvt-button--danger-secondary rvt-button--danger-secondary-focus">Button</button></td>
-            <td><button class="rvt-button rvt-button--danger-secondary rvt-button--danger-secondary-active">Button</button></td>
-            <td><button class="rvt-button rvt-button--danger-secondary" disabled>Button</button></td>
-        </tr>
-        <tr>
-            <th scope="row">Plain button</th>
-            <td><button class="rvt-button rvt-button--plain">Button</button></td>
-            <td><button class="rvt-button rvt-button--plain rvt-button--plain-hover">Button</button></td>
-            <td><button class="rvt-button rvt-button--plain rvt-button--plain-focus">Button</button></td>
-            <td><button class="rvt-button rvt-button--plain rvt-button--plain-active">Button</button></td>
-            <td><button class="rvt-button rvt-button--plain" disabled>Button</button></td>
-        </tr>
-        <tr>
-            <th scope="row">Small button</th>
-            <td><button class="rvt-button rvt-button--small">Button</button></td>
-            <td><button class="rvt-button rvt-button--small rvt-button--hover">Button</button></td>
-            <td><button class="rvt-button rvt-button--small rvt-button--small-focus">Button</button></td>
-            <td><button class="rvt-button rvt-button--small rvt-button--active">Button</button></td>
-            <td><button class="rvt-button rvt-button--small" disabled>Button</button></td>
-        </tr>
-        <tr>
-            <th scope="row">Small secondary button</th>
-            <td><button class="rvt-button rvt-button--secondary rvt-button--small">Button</button></td>
-            <td><button class="rvt-button rvt-button--secondary rvt-button--small rvt-button--secondary-hover">Button</button></td>
-            <td><button class="rvt-button rvt-button--small rvt-button--secondary rvt-button--small-focus">Button</button></td>
-            <td><button class="rvt-button rvt-button--small rvt-button--secondary rvt-button--secondary-active">Button</button></td>
-            <td><button class="rvt-button rvt-button--small rvt-button--secondary" disabled>Button</button></td>
-        </tr>
-    </tbody>
-</table>
+<button class="rvt-button
+  {{~#if type}}
+    {{~#if secondary}} rvt-button--{{type}}-{{secondary}}{{~/if}}
+  {{~/if}}
+  
+  {{~#if type}} rvt-button--{{type}}{{~/if}}
+  {{~#if secondary}} rvt-button--{{secondary}}{{~/if}}
```

Comment:
See my comment above. Just to clean up the config file and make it a bit more readable. Setting `secondary: true` in the config and then in the template here:

```handlebars
{{~# if secondary}} rvt-button--secondary{{~/if}}
```